src/Cli/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -80,8 +80,29 @@ public void Configure()
             {
                 Stopwatch beforeGenerateAspNetCertificate = Stopwatch.StartNew();
                 GenerateAspNetCertificate();
+
+                if (canShowFirstUseMessages)
+                {
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                    {
+                        // The instructions in this message only apply to Windows and MacOS.
+                        PrintFirstTimeMessageAspNetCertificate();
+                    }
+                    else
+                    {
+                        // The instructions in this message only apply to Linux (various distros).
+                        // OSPlatform.FreeBSD would also see this message, which is acceptable since we have no specific FreeBSD instructions.
+                        PrintFirstTimeMessageAspNetCertificateLinux();
+                    }
```

Comment:
I would have flipped this logic over (only check for Linux), but `OSPlatform` also contains `FreeBSD`, so I want it to fall through to the Linux message for that platform.